packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -31,7 +31,7 @@ const TEMPLATES = fs
     }
   }, {})
 const OUTPUT_PATH = path.join(getPaths().api.lib, 'auth.js')
-const WEB_SRC_INDEX_PATH = path.join(getPaths().web.src, 'index.js')
+const WEB_SRC_APP_PATH = path.join(getPaths().web.src, 'App.js')
```

Comment:
I'm not sure why we this was renamed to `App.js`?

Reply:
So `index.js` and `App.js` are separated out, so that we can render the `<App/>` without running the browser specific code in indexjs.